frame/pablo/src/lib.rs

```diff
@@ -966,6 +966,22 @@ pub mod pallet {
 			Pools::<T>::contains_key(pool_id)
 		}
 
+		fn get_price_of_lp_token(pool_id: Self::PoolId) -> Result<Self::Balance, DispatchError> {
+			let pool = Self::get_pool(pool_id)?;
+			let pool_account = Self::account_id(&pool_id);
+			match pool {
+				PoolConfiguration::StableSwap(StableSwapPoolInfo { pair, lp_token, .. }) =>
+					calculate_price_of_lp_token::<T>(pair, lp_token, &pool_account),
+				PoolConfiguration::ConstantProduct(ConstantProductPoolInfo {
+					pair,
+					lp_token,
+					..
+				}) => calculate_price_of_lp_token::<T>(pair, lp_token, &pool_account),
```

Comment:
```suggestion
				PoolConfiguration::StableSwap(StableSwapPoolInfo { pair, lp_token, .. }) |
				PoolConfiguration::ConstantProduct(ConstantProductPoolInfo {
					pair,
					lp_token,
					..
				}) => calculate_price_of_lp_token::<T>(pair, lp_token, &pool_account),
```
You can merge arms, please check the suggestion before accepting.